app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -14,12 +14,14 @@ import { BlobListFormField } from './BlobList'
 import {
   LabelList,
   LabelListEmpty,
+  ModerationLabel,
+} from '@/common/labels/List'
```

Comment:
All of these label import changes are to remove `export * from...` pattern which makes cypress/webpack choke.